### ISSUE TITLE
Allow options.sort to be a sort function

### ISSUE THREAD

**Issue by alpha-cast** [brianreavis/sifter.js#24]
This would make it consistent with the undocumented `options.score`, and would provide much simpler workarounds for some of the other issues which want to disable sorting. In that case you could just do something like `options.sort = function (results) { return results; }`.

**Comment by broerse** [comment]
We use this:

``` javascript
  sifter = new Sifter(input);
  if (!sort) {
    sifter.getSortFunction = function() {
      return null;
    };
  }
  result = sifter.search(query, options);
```

in [ember-cli-filter-by-query] (https://github.com/lazybensch/ember-cli-filter-by-query/blob/master/addon/util/filter.js)

**Comment by alpha-cast** [comment]
Yeah I did see that solution in one of the other issues. I just think it'd
be easy and clear to add this kind of thing to the API as a documented and
supported thing. People obviously want to be able to do it, and I don't
think what you put above is very obvious for a casual user of the library,
even if it's fairly easy to do.

**Comment by alpha-cast** [comment]
Hmm. Well, maybe it is reasonably obvious and I'm just being pedantic 😜

**Comment by broerse** [comment]
@alpha-cast Many people searched for it and didn't find it so I don't think it is to obvious. So perhaps you are not pedantic :smile: It would be great if you can create a PR for the documentation.

**Comment by alpha-cast** [comment]
Yeah OK. I've never actually created a pull request for GitHub before, but have been meaning to learn for a while. Better late than never ;-P

**Comment by broerse** [comment]
Super! If you need help with github please let me know.
